### PR TITLE
Add example with Cypress Plugin

### PR DESCRIPTION
**Pull request by FriggaHel**

### CODE REVIEW

File: cypress/cypress.config.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'cypress';
+import CypressVisualPlugin from '@saucelabs/cypress-visual-plugin';
+
+export default defineConfig({
+  e2e: {
+    saucelabs: {
+      buildName: 'SauceDemo - Cypress',
+      namePrefix: 'visual: ',
```
Comment (ebekebe) [inline cypress/cypress.config.ts:8]: The `namePrefix` isn't explained and for the sake of simplicity I would leave it out in the example.

File: cypress/cypress/e2e/saucedemo.cy.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+context('Sauce Demo', () => {
+  beforeEach(() => {
+  })
```
Comment (ebekebe) [inline cypress/cypress/e2e/saucedemo.cy.ts:3]:
```suggestion
```

File: cypress/cypress/e2e/saucedemo.locked.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+context('Sauce Demo', () => {
+  beforeEach(() => {
+  })
+
```
Comment (ebekebe) [inline cypress/cypress/e2e/saucedemo.locked.ts:4]:
```suggestion
```

File: cypress/cypress/e2e/saucedemo.standard.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+context('Sauce Demo', () => {
+  beforeEach(() => {
+  })
+
```
Comment (ebekebe) [inline cypress/cypress/e2e/saucedemo.standard.ts:4]:
```suggestion
```

File: cypress/cypress/support/e2e.ts
Comment (ebekebe) [inline cypress/cypress/support/e2e.ts:1]: Is this empty file required?

Comment (FriggaHel) [reply cypress/cypress/support/e2e.ts:1]: You can remove it, but that involves a specific setting in cypress.
This is more common to keep it empty to keep the config by default. (In a bigger project you will have a need for it anyway).

File: cypress/cypress.config.ts (hunk 1 above)
Comment (FriggaHel) [reply cypress/cypress.config.ts]: Agreed ! I removed it !

**Review by kb-kerem — APPROVED** [review]

**Pull request merged**